azurelinuxagent/pa/deprovision/default.py

```diff
@@ -95,6 +104,9 @@ def del_dhcp_lease(self, warnings, actions):
         actions.append(DeprovisionAction(fileutil.rm_files, ["/var/db/dhclient.leases.hn0",
                                                              "/var/lib/NetworkManager/dhclient-*.lease"]))
 
+        # For OpenBSD
+        actions.append(DeprovisionAction(fileutil.rm_files, ["/var/db/dhclient.leases.hvn0"]))
```

Comment:
I modeled it after the FreeBSD one above that only deletes hn0 (I don't know what NetworkManager is). But, yes, that should be OK to remove /var/db/dhclient.leases.hvn* in case that there was any additional configured NIC.